src/tools/exec/bir_exec_expLib.sml

```diff
@@ -181,74 +174,54 @@ val UPDATE_EQ_num = prove(``
   SIMP_TAC std_ss [UPDATE_EQ]
 );
 
+fun combin_purge_conv ct =
```

Comment:
This isn't directly related to this PR, but can you add a comment explaining what this functions does (as it is a public function of this lib)? Ideally, write a `.sig` file with short comments :slightly_smiling_face:

Please read the next comment line 221. Then, if you want to keep this function, you can just write a comment in this file as I just created a new issue for this: #34

Reply:
I would like to take this separately to resolve the issue you created for the hole repository or stepwise big parts of it.